docs/akamai/akamai-sdk.md

```diff
@@ -45,6 +45,39 @@ async function main() {
 main();
 ```
 
+</TabItem>
+<TabItem value="python" label="Python SDK">
+
+- Step 1. [Install the SDK](../../docs/sdks/install-sdks.md).
```

Comment:
@dlozina-macrometa , remind me to show you how to single source this so you don't have to keep copy/pasting or writing this out.